english/invite-friends/camp_fire/abduction/divide_conquer/badCop.md

```diff
@@ -0,0 +1 @@
+You remember your intensive, highly secretive training as you look at this shifty individual.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.